### ISSUE TITLE
train set, val set and test set

### ISSUE THREAD

**Issue by DUAT-Maker** [jeonsworld/ViT-pytorch#60]
Hi, perhaps, I didn't understand your code clearly, but I think you didn't split the dataset into train set, val set and test set. You straightly validate and test your model on test set. If that is right, I think it's wrong.

**Comment by DUAT-Maker** [comment]
Can you provide the code for training Imagenet, thank you.

**Comment by abhishektyaagi** [comment]
Hi @Anonymity111222 ,
Were you able to figure out how to train the ViT-base models here on ImageNet?
